cmd/lakectl/cmd/fs.go

```diff
@@ -28,7 +28,12 @@ Size: {{ .SizeBytes }} bytes
 Human Size: {{ .SizeBytes|human_bytes }}
 Physical Address: {{ .PhysicalAddress }}
 Checksum: {{ .Checksum }}
-Content-Type: {{ .ContentType }}
+Content-Type: {{ .ContentType }} {{ if $.Metadata }}
```

Comment:
1. You will probably need to fix the integration tests (esti)
2. Consider checking also if .Metadata.AdditionalProperties holds something in this level

Reply:
1. Yep, fixing it now.
2. Can't use `{{-` because it will eat ALL previous whitespace, including the newline at the end of the output in case there's no metadata.
3. Done!